arwen/errors.go

```diff
@@ -23,6 +23,9 @@ var ErrSignalError = errors.New("error signalled by smartcontract")
 // ErrExecutionFailed signals that the execution failed
 var ErrExecutionFailed = errors.New("execution failed")
 
+// ErrExecutionFailedWithTimeout signals that the execution failed with timeout
+var ErrExecutionFailedWithTimeout = errors.New("execution failed")
```

Comment:
might rename to `execution timed out` as discussed internally

Reply:
done